docs/writing-tests/file-names.md

```diff
@@ -27,6 +27,7 @@ themselves precede any test type flag, but are otherwise unordered.
 
 
 `.https`
+`.serviceworker`
```

Comment:
I think this kind of implies that `.serviceworker` is a thing that you can/should add by hand. I think maybe we shouldn't encourage that but should just document it in the documentation for `any` tests and note that it implies https.

Reply:
Ah, I guess the recommendation for normal service worker tests is to still use .https then?

Reply:
Yes, `.serviceworker` is for the case where you want the automagically generated HTML wrapper for the `.js` file.

Reply:
Right, and only `any.js` so far.